lambda-notes/src/Views/EditNoteContainer.js

```diff
@@ -0,0 +1,280 @@
+import React from 'react'
+import styled from 'styled-components';
+import { connect } from "react-redux";
+import { editNote,deleteNote } from '../Actions';
+import Modal from "../Components/Modal";
+import NavSection from '../Components/NavView/NavSection'
+
+
+const ModalDiv = styled.div`
+    background-color: rgba(0,0,0,0.5);
+    position: fixed;
+    height: 100%;
+    width: 100%;
+    top: 0;
+    left: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+`;
+const ModalContainer = styled.div`
+    background:white;
+    border:1px solid #ccc;
+    min-height:190px;
+    min-width:530px;
+    border-radius:5px;
+    display:flex;
+    flex-direction:column;
+    justify-content:space-around;
+    align-items:center
+`;
+const Loading = styled.h1`
+    font-size:4rem;
+`
+const ActionsDiv = styled.div`
+    display:flex;
+    justify-content:flex-end;
+`
+const WrapperDiv = styled.div`
+    margin-top:10px;
+    border:1px solid #CECECE;
+    display:flex;
+`
+const NoteSection = styled.div`
+    width:75%;
+    background-color: #F2F1F2;
+    padding-left:25px;
+    padding-right:25px;
+    padding-top:25px;
+`
+const NavButton = styled.div`
+    background-color:#2FB8BC;
+    width:150px;
+    height:35px;
+    color:white;
+    display:flex;
+    justify-content:center;
+    align-items:center;
+`
+const FormSection = styled.div`
+    padding-left:25px;
+    padding-right:25px;
+
+`
+
+const Form = styled.form`
+    display:flex;
+    flex-direction:column;
+`
+
+const TitleInput = styled.input`
+    width:355px;
+    height:43px;
+`
+const TextArea = styled.textarea`
+    margin-top:25px;
+    width:610px;
+    height:350px;
+`
+const SaveButton = styled.button`
+    width:190px;
```

Comment:
a lot of these buttons are similar. Would be worth refactoring a single Button component, and then composing specific versions of it, possibly

Reply:
that way, if there is a design change in buttons overall, you don't have to go and change every button's property